Jenkinsfile

```diff
@@ -0,0 +1 @@
+buildPlugin()
```

Comment:
You need write permissions to get it running

Reply:
That's what I figured, but I thought I could add it for when it was merged to master.